src/components/Maps/AK_Fires.vue

```diff
@@ -72,13 +78,259 @@ export default {
           'abstract': 'This layer shows historical fire perimeters from 1940-2016.  _More recent wildfires often stop fires from spreading due to the lack of fuel, but does this always hold true?_\n\nTo access and learn more about this dataset, visit the [AICC](https://fire.ak.blm.gov).\n',
           'name': 'geonode:fireareahistory',
           'title': 'Historical extent, 1940-2016'
+        },
+        {
+          'name': 'fires_2017',
+          'title': 'All fires, 2017',
+          'object': () => {
+            console.log('in object thing, this: ', this)
```

Comment:
Should I give these `console.log()` statements a pass since this is still a work in progress?

Reply:
Nope, el deleto grande incoming!